sites/friday/src/get-started/laravel/deploy/scheduling.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Workers, cron jobs, and task scheduling"
+sidebarTitle: "Scheduling tasks"
+weight: -100
+description: Schedule tasks for your Laravel app.
+---
+
+Laravel offers a very convenient and flexible way of scheduling tasks. A large set
+of [helper functions](https://laravel.com/docs/scheduling#schedule-frequency-options)
+help you schedule commands and jobs.
+
+Once the scheduled tasks are defined, they need to be effectively executed at the
+right time and pace. The recommended way is a cron configuration entry running
+the `artisan schedule:run` command.
+
+```yaml {configFile="app"}
```

Comment:
I think we need to make it clear that the cron conf should be added under the `applications` top-level key.

Reply:
concur.